### ISSUE TITLE
Issue invoking GFPGAN

### ISSUE THREAD

**Issue by thelamedia** [lstein/stable-diffusion#370]
After preloading the models and running dream.py, I'm getting an error when trying to utilize GFPGAN.  It says that --gfpgan needs to be invoked, however when launching dream.py that doesn't seem to be an option anymore.  Even when using the gfpgan_bg_upsampler and tile flags, it doesn't seem to invoke.  Am I doing something wrong?

**Comment by blessedcoolant** [comment]
> After preloading the models and running dream.py, I'm getting an error when trying to utilize GFPGAN. It says that --gfpgan needs to be invoked, however when launching dream.py that doesn't seem to be an option anymore. Even when using the gfpgan_bg_upsampler and tile flags, it doesn't seem to invoke. Am I doing something wrong?

`--gfpgan` is not an option anymore because it gets enabled by default. All you need to do is set the `--gfpgan_dir` during boot to point to the folder where your GFPGAN is located. After which you can use `-G` and `-U` in the prompt commands to invoke GFPGAN and Real-ESRGAN respectively. There's more info in the documentation.

Make sure you are on the latest version of the code.

**Comment by thelamedia** [comment]
Ah got it.  I'll give that a shot.  So it no longer defaults to having it as a sister directory and now requires manual linking?  This makes sense.  I hadn't tried this yet.  Thanks!

**Comment by blessedcoolant** [comment]
> Ah got it. I'll give that a shot. So it no longer defaults to having it as a sister directory and now requires manual linking? This makes sense. I hadn't tried this yet. Thanks!

It does. The default directory is still a sister directory called GFPGAN. But if your GFPGAN is not setup in the place, then you need to manually link it.

**Comment by thelamedia** [comment]
Interesting.  I do have it set up in a sister directory under that name and it doesn't default there.  I'll try a manual link to see if this makes a difference.

**Comment by kaishin** [comment]
I am also unsure why the upscaling is not taking place. I launched the script using this command:

```
python3 scripts/dream.py --gfpgan_dir ../GFPGAN/
```

I followed the instructions from the GFPGAN repo, using the same conda env (ldm), and downloaded the pretrained model.
Both this repo and and `GFPGAN` are cloned in  the same parent repo.

Running the dream commands with `-U 2` has no effect (no upscaled images in the outputs folder).

Not sure if it's the same issue as @thelamedia , but anyone has any pointers?
